server/database/sql/create_tables.sql

```diff
@@ -35,6 +35,20 @@ create table if not exists tracker (
     foreign key("user") references "user"(id)
 );
 
+create table if not exists word (
+    id serial primary key,
+    word varchar (50) not null,
```

Comment:
is varchar (50) the max number of characters allowed in the word?

Reply:
Yup 50 max. Should it be higher? The whole post has a 512 max